azurelinuxagent/common/version.py

```diff
@@ -25,6 +25,35 @@
 from azurelinuxagent.common.utils.flexible_version import FlexibleVersion
 from azurelinuxagent.common.future import ustr, get_linux_distribution
 
+__DAEMON_VERSION_ENV_VARIABLE = '_AZURE_GUEST_AGENT_DAEMON_VERSION_'
```

Comment:
Why not put this under this class - 

`class ExtCommandEnvVariable(object):`

If position of that class is a problem, you can also move it from `exthandlers.py` to some file under `common/` maybe?

Reply:
those are mean for extensions, this one is not

i used underscores to mark variables not meant to be used by extensions